test/ably/rest/restcrypto_test.py

```
@@ -79,16 +79,16 @@ async def test_crypto_publish(self):
         message_contents = dict((m.name, m.data) for m in messages)
         log.debug("message_contents: %s" % str(message_contents))
 
-        assert "This is a string message payload" == message_contents["publish3"],\
+        assert "This is a string message payload" == message_contents["publish3"], \
```

Ruff (0.11.9)

Yoda condition detected

(SIM300)